### PR TITLE
Ifteda About Page 2nd Draft

### PR DESCRIPTION
**Pull request by ifteda**
- still not sure what visual we want in the first section. added the BLM logo for now
- left align all other sections or align with the first section? aligning with the first section creates some empty space to the left
- Contact Us section: use an email icon? remove hackbeanpot.com?

screencapture-localhost-3000-about-1593809293721

### CODE REVIEW

File: src/components/AboutPage.js [hunk 1]
```diff
@@ -59,74 +62,87 @@ const AboutPage = () => (
           <li>...</li>
           <li>...</li> */}
         </ul>
+        <p>
+          Contact us if you would like to get involved.
```
Comment (nupur-n) [inline src/components/AboutPage.js]: Could you change this to something like: Contact us if you would like to get involved. We are always looking to partner with organizations who are committed to being anti-racist and want to help work for justice and equity through tech. You can also join our slack workplace [include link] to learn more from the organizing team.

**Review by nupur-n — CHANGES_REQUESTED** [review]
Most of it looks good to me! Added one comment about wording to help the section look a little more filled. 

Spacing wise, I think we could utilize some of Shania's icons used elsewhere in here as well. Can discuss further or at meeting Sunday but draft itself looks set to me.

File: src/styling/about.css [hunk 2]
```diff
@@ -28,4 +32,11 @@ ul {
 .container {
   margin-top: 100px;
   margin-bottom: 100px;
+}
+
+.socialLogo {
+  height: 30px;
+  width: 30px;
```
Comment (liu-caro) [inline src/styling/about.css]: Could you remove this line of styling? It's making the LinkedIn logo look squished and the images look okay without it.

File: src/components/AboutPage.js [hunk 3]
```diff
@@ -59,74 +62,87 @@ const AboutPage = () => (
...
-        <a href="https://www.linkedin.com/company/hackbeanpot-inc">
-          LinkedIn
-        </a> <br />
-        <a href="https://twitter.com/hackbeanpot">
-          Twitter
         </a>
+        <br />
+        <a href="https://hackbeanpot.com/"
+        target="_blank" rel="noopener noreferrer">
+          hackbeanpot.com
```
Comment (liu-caro) [inline src/components/AboutPage.js]: Could we add text around these maybe? (we can try to fix how links show up in a later style runthrough)
Email us at <email>
Visit us at <website>

Comment (ifteda) [reply src/components/AboutPage.js]: same line or preceding line?

**Comment by ifteda** [comment]
screencapture-localhost-3000-about-1593809293721

**Comment by nupur-n** [comment]
Looks good!

**Review by nupur-n — APPROVED** [review]

**Review by liu-caro — APPROVED** [review]

**Pull request merged**